src/firebase.js

```diff
@@ -0,0 +1,17 @@
+const { initializeApp } = require('firebase/app');
+const { ref, uploadBytes, getStorage } = require('firebase/storage');
+
+const firebaseConfig = {
+  apiKey: process.env.FIREBASE_API_KEY,
+  authDomain: process.env.FIREBASE_AUTH_DOMAIN,
+  projectId: process.env.FIREBASE_PROJECT_ID,
+  storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
+  appId: process.env.FIREBASE_APP_ID,
+};
+
+const firebaseApp = initializeApp(firebaseConfig);
+const firebaseStorage = getStorage();
```

Comment:
It's up to you, but one thing we could do is call 

```suggestion
const firebaseStorage = getStorage(firebaseApp);
```

here and then export it; and then in the controller, we can just use `firebaseStorage` without having to do the step of "get storage" for each request. But I'm not sure if that really optimizes much (that is, I'm not sure if the library is making a full HTTP call to Firebase to get this information or what exactly); so I'm fine leaving the call in the controller as-is otherwise.